community-containers/helloworld/container-definition.md

```diff
@@ -0,0 +1,68 @@
+| `shm_size`                | TODO                                                                                                                                                                  |
+| `secrets`                 | TODO                                                                                                                                                                  |
+| `ui_secret`               | TODO                                                                                                                                                                  |
+| `devices`                 | TODO                                                                                                                                                                  |
+| `enable_nvidia_gpu`       | TODO                                                                                                                                                                  |
+| `apparmor_unconfined`     | TODO                                                                                                                                                                  |
+| `backup_volumes`          | List of volumes should be included in the AIO backup.                                                                                                                 |
+| `nextcloud_exec_commands` | TODO                                                                                                                                                                  |
+| `profiles`                | TODO                                                                                                                                                                  |
+| `read_only`               | TODO                                                                                                                                                                  |
+| `init`                    | TODO                                                                                                                                                                  |
+| `tmpfs`                   | TODO                                                                                                                                                                  |
+| `volumes`                 | List of volumes to mount in the container. See [volumes section](#volumes)                                                                                            |
+
+### Ports
+
+| Field         | Description                         |
+|---------------|-------------------------------------|
+| `ip_binding`  | The IP address to bind the port to. |
+| `port_number` | The port number to expose.          |
+| `protocol`    | The protocol to use.                |
+
+### Healthcheck
```

Comment:
We should probably state in the docs that the options here are not considered inside aio. Same for other things like profiles and user...

Reply:
So these options need to be set in the Dockerfile of the container...